operator-rhel.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hazelcast-operator
+  labels:
+    app.kubernetes.io/name: hazelcast-operator
+    app.kubernetes.io/instance: hazelcast-operator
+    app.kubernetes.io/managed-by: hazelast-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: hazelcast-operator
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: hazelcast-operator
+        app.kubernetes.io/instance: hazelcast-operator
+        app.kubernetes.io/managed-by: hazelcast-operator
+      annotations:
+        productID: hazelcast-operator
+        productName: Hazelcast Operator
+        productVersion: 0.2.3
+    spec:
+      serviceAccountName: hazelcast-operator
+      securityContext:
+        capabilities:
+          drop:
+          - ALL
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: beta.kubernetes.io/arch
+                operator: In
+                values:
+                - amd64
+      containers:
+        - name: hazelcast-operator
+          image: registry.connect.redhat.com/hazelcast/hazelcast-operator:0.2.3
+          imagePullPolicy: Always
+          env:
+            - name: WATCH_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: OPERATOR_NAME
+              value: "hazelcast-operator"
+            - name: RELATED_IMAGE_HAZELCAST
+              value: registry.connect.redhat.com/hazelcast/hazelcast-4-rhel8:4.0-1
+            - name: RELATED_IMAGE_MANCENTER
+              value: registry.connect.redhat.com/hazelcast/management-center-4-rhel8:4.0-2
```

Comment:
Can we somehow extract environment variables and pass to kubectl? And it readme, we can point how people can find latest hazelcast images...

That way we don't have to update operator.yaml files after every hazelcast release. WDYT?

Reply:
There is no such mechanism. You can only use variables in Helm Chart (not applicable here) or in the OpenShift templates (but I see they are not well adopted, so I don't want to use them here).

We could obviously use `sed`, but I don't want to use it here neither.